src/main/java/com/splunk/logging/HttpEventCollectorSender.java

```diff
@@ -155,8 +157,48 @@ public synchronized void send(
         // create event info container and add it to the batch
         HttpEventCollectorEventInfo eventInfo =
                 new HttpEventCollectorEventInfo(severity, message, logger_name, thread_name, properties, exception_message, marker);
+        queueEventInfo(eventInfo);
+    }
+
+    /**
+     * Send a single logging event including the full information from the throwable.
+     *
+     * Contrast with the alternate send method that only sends the throwable's message.
+     */
+    public synchronized void send(
+            final String severity,
+            final String message,
+            final String logger_name,
+            final String thread_name,
+            Map<String, String> properties,
+            final Throwable throwable,
+            Serializable marker
+    ) {
+        // create event info container and add it to the batch
+
+
+        HttpEventCollectorEventInfo eventInfo =
+                new HttpEventCollectorEventInfo(
```

Comment:
nit: I'd like to see `new HttpEventCollectorEventInfo(` on the same line as the assignment